frontend/rolecall/src/app/piece/piece_editor.component.scss

```diff
@@ -261,3 +304,7 @@
 #position-pool.cdk-drop-list-dragging .position-container:not(.cdk-drag-placeholder) {
   transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
 }
+
+.rcmitem {
```

Comment:
What's `rcm` here? Should it be `rcm-item`? Or is there a more descriptive name possible?

Reply:
Done.